src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -266,8 +266,9 @@ def fall_detect(self, image=None):
                     if angle_change > self._fall_factor:
                         # insert a box that covers the whole image as a workaround
                         # to meet the expected format of the save_detections element
-                        box = [0, 0, 1, 1]
-                        inference_result.append(('FALL', pose.score, box, angle_change))
+                        #box = [0, 0, 1, 1]
+                        #inference_result.append(('FALL', pose.score, box, angle_change))
+                        inference_result.append(('FALL', pose.score, angle_change))
```

Comment:
Let's also store the before and after images along with their keypoints to help debug fall detection issues (such as false positives). We may also end up rendering in the UI before and after image details with a fall vector to make it clear to the user WHY the AI detected a fall. It would also give the user a richer context of WHY and HOW the person fell.

Reply:
@bhavikapanara fyi ^^^ Expecting your thoughts.

Reply:
@ivelin 
Ok...can you please list down all items that you want to add in `inference_result`?
Will solve the rest of the comments that are related to `inference_result` once I have a format of `inference_result`

Reply:
Ok, this seems more complicated than it needs to be. Let's keep things simple.
Populate the bounding box as described and do not add more data attributes in this PR. We will handle that in a separate issue and PR.
For output score value, use the average of the hip&shoulder scores that were actually used for angle calculation. It's a reasonable score metric to work with for now until we know that something better is needed.